README.md

```diff
@@ -88,6 +88,16 @@ workers:
   transfer: 10
 ```
 
+Excluding directories or files to be scraped from the source can be reached with specifying a regex in the job definition:
+```yaml
+jobs:
+  - from: http://de.archive.ubuntu.com/ubuntu/
+    to:   mirror/ubuntu-repos
+    excl: /sub_dir$|.gz$
+```
+
+This would exclude directories named `sub_dir` and files with extension `gz` on every level.
```

Comment:
Actually, at this point, directory URLs have a trailing slash (which is actually valuable since it allows for control over whether the regex matches a file or directory), so the correct expression would be `/subdir/$|.gz$`.